feelslikemonday/app/src/main/java/com/example/feelslikemonday/ui/followrequest/FollowerRequestFragment.java

```diff
@@ -14,6 +14,7 @@
 import androidx.lifecycle.ViewModelProviders;
 
 import com.example.feelslikemonday.R;
+import com.example.feelslikemonday.model.MoodEvent;
```

Comment:
In this fragment I don't use the MoodEvent and won't need it, but it's good to have here.

Reply:
ok sounds good